ui/apps/platform/src/Components/CompoundSearchFilter/components/EntitySelector.tsx

```diff
@@ -33,6 +33,7 @@ function EntitySelector({ selectedEntity, onChange, config }: EntitySelectorProp
             onChange={onChange}
             ariaLabelMenu="compound search filter entity selector menu"
             ariaLabelToggle="compound search filter entity selector toggle"
+            isDisabled={entities.length === 1}
```

Comment:
Similar to the attribute selector, I disable the entity selector when only one entity is provided

Reply:
See comment #11431

Reply:
That's a valid concern. I'll double-check with UX if the disabled dropdown might be confusing. If they agree, maybe we can just enable it even if there's only one option.